support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -174,10 +171,9 @@ test.describe('Subscribe (S+) incl PromoCode via the Tiered checkout', () => {
 			// Thank you
 			await expect(
 				page.getByText(testDetails.expectedThankYouText).first(),
-			).toBeVisible();
+			).toBeVisible({ timeout: 600000 });
```

Comment:
We actually wait on the text being visible rather than the URL being the right one.